lib/components/Cards/ProductCard.tsx

```diff
@@ -19,7 +17,7 @@ export default function ProductCard({ id, title, image, description, price }: Pr
       key={id}
       className="flex flex-col justify-around m-7 bg-slate-100 p-5 rounded-md text-ellipsis"
     >
-      <Image src={image} alt={description} layout="fill" className="h-60 w-80 object-contain" />
```

Comment:
Next.js recommends using <Image/> component for images because of performance optimizations. You can checkout the API in below resource:

**Resource**
https://nextjs.org/docs/api-reference/next/image

Reply:
Resolved